pygaggle/data/convert_duot5_output_to_msmarco_run.py

```diff
@@ -42,7 +42,7 @@ def load_run(path):
     print('Sorting candidate docs by rank...')
     sorted_run = collections.OrderedDict()
     for query_id, doc_titles_ranks in tqdm(run.items()):
-        sorted(doc_titles_ranks, key=lambda x: x[1])
+        doc_titles_ranks.sort(key=lambda x: x[1])
```

Comment:
awesome!

Reply:
Can't believe we let this pass for so long hahaha